nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -388,6 +388,23 @@ def _impl(inputs, attr, params):
 
     return _impl
 
+def _slice():
+    def _impl(inputs, attr, params):
+        begin = params.pop(inputs[1].list_output_names()[0]).asnumpy().tolist()
+        size = params.pop(inputs[2].list_output_names()[0]).asnumpy().tolist()
+        data_shape = attr['_input_shapes'][inputs[0]]
+        data_dim = len(data_shape[0])
+        end = size
+        for i in range(data_dim):
+            if size[i] == -1:
+                end[i] = data_shape[0][i] - begin[i]
+            else:
+                end[i] += begin[i]
+        out = _sym.strided_slice(inputs[0], begin=begin, end=size)
+        out_shape = _infer_out_shapes(out, params)[0]
+        return _sym.reshape(out, shape=tuple(out_shape))
```

Comment:
Do we need this reshape ?
Is strided_shape output is not in expected shape ?

Reply:
Checked this. We don't need reshape after strided_slice.Thanks!